microprofile.ls/README.md

```diff
@@ -112,3 +112,26 @@ To add external snippets (like Quarkus snippets) an implementation of `ISnippetR
  * [Java Quarkus snippets loader](https://github.com/redhat-developer/quarkus-ls/tree/master/quarkus.ls.ext/com.redhat.quarkus.ls/src/main/java/com/redhat/quarkus/snippets).
  * [JSON Quarkus snippet](https://github.com/redhat-developer/quarkus-ls/tree/master/quarkus.ls.ext/com.redhat.quarkus.ls/src/main/resources/com/redhat/quarkus/snippets).
  * Java Quarkus snippets loader must be declared in [META-INF/services/org.eclipse.lsp4mp.ls.commons.snippets.ISnippetRegistryLoader](https://github.com/redhat-developer/quarkus-ls/blob/master/quarkus.ls.ext/com.redhat.quarkus.ls/src/main/resources/META-INF/services/org.eclipse.lsp4mp.ls.commons.snippets.ISnippetRegistryLoader) 
+
+ Manage complex properties
+-------
+
+The available properties in `microprofile-config.properties` comes from the external component (ex: MicroProfile JDT LS extension). In some case a property 
```

Comment:
The properties available in `microprofile-config.properties` come from ...

Reply:
fixed.